pandas/core/internals/managers.py

```diff
@@ -1649,6 +1640,7 @@ def concat(self, to_concat, new_axis: Index) -> "SingleBlockManager":
         -------
         SingleBlockManager
         """
+        assert isinstance(new_axis, Index), new_axis
```

Comment:
this is already asserted in the SingleBlockManager constructor, so is not needed here?